tests/multicomponent_averaging/solution/solution-00000.0000.gnuplot

```diff
@@ -0,0 +1,91 @@
+# This file was generated by the deal.II library.
+
+
+#
+# For a description of the GNUPLOT format see the GNUPLOT manual.
+#
+# <x> <y> <velocity> <velocity> <p> <T> <C_1> <density> <thermal_expansivity> <specific_heat> <viscosity> <thermal_conductivity> 
+6.371e+06 0 0 0 3.78097e+07 273 0.5 2476.34 2.39692e-05 1500 2.39692e+20 2.39692 
```

Comment:
Thank you for spotting this! This is a significant bug.

The equation of state has been broken at some time after I made it. The reference temperature for the equation of state has been made a function of adiabatic heating. This is wrong; the reference temperature is a fixed constant in this equation of state. 

The adiabatic gradient should be determined by the equation of state, not the other way round. 
For now, I've returned the EoS to its intended form, but do we need to discuss the interaction with the adiabatic heating plugin?

Reply:
Thank you for spotting this! This is due to changes in multicomponent incompressible, which will be changed further in #3961. There's nothing wrong with this output; the behaviour is unexpected because the new functionality is not yet documented. Again, this is rectified in #3961.

Reply:
But the test output should reflect what your comment in the prm file says. You say that at the reference temperature of 273 , the density should be 2400, but it's not. Wouldn't it make more sense to make this a test without adiabatic heating (so the problem doesn't occur), and then make a separate test for the new functionality in #3961?